### PR TITLE
Add link to docs for cron format

### PR DESCRIPTION
**Pull request by pierlo-upitup**
image

@sascala & @leemunroe any comments?

Edit (@mlunoe): Updated image to reflect the recruirements from @sascala, and worked with @joel-hamill to finalize it with your suggestions.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @MatApple

File: src/js/schemas/job-schema/Schedule.js [hunk 1]
```diff
@@ -35,7 +35,11 @@ const General = {
     },
     cron: {
       title: 'CRON Schedule',
-      description: 'Enter your schedule in CRON format, e.g. 0 20 * * *',
+      description: (
+        <span>
+          Enter your schedule in CRON format, e.g. <i>0 0 20 * * *</i>. <a href="https://dcos.io/docs/1.8/usage/jobs/getting-started" target="_blank">Read the docs</a> for more info.
```
Comment (wavesoft) [inline src/js/schemas/job-schema/Schedule.js]: I would suggest a `<br/>` after the first description and putting the `<a>` in a `<small>` in order to differentiate style.

Comment (mlunoe) [inline src/js/schemas/job-schema/Schedule.js]: Done

Comment (mlunoe) [inline src/js/schemas/job-schema/Schedule.js]: @pierlo-upitup @wavesoft: FYI: We need to make sure to not include links with versions in them, we have `/latest` for that, which redirects you to the latest docs. Also, we have `MetadataStore.buildDocsURI` function that will make sure to distinguish open vs. enterprise

**Comment by mlunoe** [comment]
**mesosphere-ci** Retest this please

File: src/js/schemas/job-schema/Schedule.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import MetadataStore from '../../stores/MetadataStore';
```
Comment (jfurrow) [inline src/js/schemas/job-schema/Schedule.js]: This should be below the `React` import.

**Comment by jfurrow** [comment]
Other than the import, this is good to go!

**Comment by mlunoe** [comment]
Thanks for noting! Created a follow up PR here: https://github.com/dcos/dcos-ui/pull/914

**Pull request merged**